packages/design-system/src/MetaFilterSelect.tsx

```diff
@@ -151,16 +151,20 @@ const SelectValueContainer: React.FC<
   React.ComponentProps<typeof SelectComponents.ValueContainer>
 > = (props) => {
   const {
-    selectProps: { value, title, menuIsOpen },
+    selectProps: { value, title: selectTitle, menuIsOpen },
   } = props;
 
   let tagLabel = '';
-  if (Array.isArray(value) && value.length > 0) {
+  if (value.length > 0) {
     tagLabel = value.length.toString();
   }
-  if (value && !Array.isArray(value)) {
-    tagLabel =
-      title.toLowerCase() === 'availability' ? `>${value.value}` : value.value;
+  let title = selectTitle;
+  if (title.toLowerCase() === 'availability' && value.length > 0) {
```

Comment:
This sort of logic should probably not be in a component in the `design-system` package as those components are intended to be generic and reusable, no?  Could you e.g. pass in the value of `tagLabel` instead?